scripts/gen-dockerfile.sh

```diff
@@ -78,18 +70,18 @@ done
 
 # Create dummy files, build the dependencies
 # then remove TiKV fingerprint for following rebuild.
-# Finally, remove test dependencies and profile features.
+# Finally, remove fuzz and profile features.
 cat <<EOT
 RUN mkdir -p ./cmd/src/bin && \\
     echo 'fn main() {}' > ./cmd/src/bin/tikv-ctl.rs && \\
     echo 'fn main() {}' > ./cmd/src/bin/tikv-server.rs && \\
     for cargotoml in \$(find . -name "Cargo.toml"); do \\
         sed -i '/fuzz/d' \${cargotoml} && \\
-        sed -i '/test\_/d' \${cargotoml} && \\
-        sed -i '/profiler/d' \${cargotoml} && \\
-        sed -i '/\"tests\",/d' \${cargotoml} ; \\
-    done && \\
-    make build_dist_release
+        sed -i '/profiler/d' \${cargotoml} ; \\
```

Comment:
Why not update tests?

Reply:
It mistakenly removes engine_trait/tests's package name("test_engine_traits"), and cargo build to parse Cargo.toml. Furthermore, `make build_dist_release` does not build tests crates anyway.